backend/entities/Models/LeasedLandMetadata.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace Pims.Dal.Entities.Models
+{
+    public class LeasedLandMetadata
+    {
+        #region Properties
+        /// <summary>
+        /// get/set - Whether an exemption was requested for the ERP.
+        /// </summary>
+        public string OwnershipNote;
```

Comment:
because we only have the option to own the parcel or other, I'm not seeing the need for additional fields at this time.

Reply:
This `metadata` actually needs to be an array of notes.  It is possible the building exists on multiple parcels that are not in PIMS inventory and are leased or owned by external entities.

Reply:
oh, but it isn't possible to enter more then one in the UI at the moment (only one note per associated parcel), and this metadata only exists on the building -> parcel join table.